src/Microsoft.AspNetCore.Routing.Abstractions/IOutboundParameterTransformer.cs

```diff
@@ -6,13 +6,13 @@ namespace Microsoft.AspNetCore.Routing
     /// <summary>
     /// Defines the contract that a class must implement to transform parameter values.
     /// </summary>
-    public interface IParameterTransformer : IParameterPolicy
+    public interface IOutboundParameterTransformer : IParameterPolicy
     {
         /// <summary>
         /// Transforms the specified parameter value.
         /// </summary>
         /// <param name="value">The parameter value to transform.</param>
         /// <returns>The transformed value.</returns>
-        string Transform(string value);
+        string Transform(object value);
```

Comment:
Yeah, I debated this. 

I think we should make a decision now since it will require cross-repo changes. How about `TransformInbound` and `TransformOutbound`? for names?